### DISCUSSION TITLE
Run test failed with  SIGSEGV (Segmentation fault)

### DISCUSSION THREAD

**Discussion by luhc228** [vitest-dev/vitest#6451] [Q&A]
Now I want to require a binary file in my test file: 

```js
import n = require('./darwin/Release/testing.node')

console.info( n)
```
But I get the following error(vitest 1.x):
```md
Command was killed with SIGSEGV (Segmentation fault): vitest basic.test.ts
```

I get the following error(vitest 2.x):

<img width="1062" alt="image" src="https://github.com/user-attachments/assets/47ca8300-444f-4646-a2f2-c6337faa1ab1">

Maybe my binary file will cause the memory problem in child_process or worker_thread. Are there any solutions such as don't run in child_process or worker_thread?

**Comment by sheremet-va** [comment]
There is no way to run tests in the same process.

**Reply by luhc228** [reply]
Thanks. @sheremet-va I think it's my problem. I have solved it.
